modules/text2vec-upstage/clients/meta.go

```diff
@@ -0,0 +1,19 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clients
+
+func (v *vectorizer) MetaInfo() (map[string]interface{}, error) {
+	return map[string]interface{}{
+		"name":              "Upstage Module",
+		"documentationHref": "https://api.upstage.ai/v1/solar/embeddings",
```

Comment:
Can we change this link to: https://developers.upstage.ai/docs/getting-started/quick-start? the one provided doesn't lead to API documentation